CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 ### Bug fixes
 
 - [PBD] Obfuscate cifs password from device config [#4384](https://github.com/vatesfr/xen-orchestra/issues/4384) (PR [#4401](https://github.com/vatesfr/xen-orchestra/pull/4401))
+- [Backup restore & Replication] Don't copy `sm_config` to new VDIs which might leads to broken coalesces [#4482](https://github.com/vatesfr/xen-orchestra/issues/4482) (PR [#4484](https://github.com/vatesfr/xen-orchestra/pull/4484))
```

Comment:
@olivierlambert Is it correct and clear enough?

Reply:
Well :thinking: Technically the coalesce is not broken, I would said "which might leads to extra coalesce".

"Extra" or "useless" coalesce.

Reply:
Done.